### PR TITLE
Provide a gtestWrapper header to skip useless warnings

### PR DESCRIPTION
**Pull request by piponazo**
In this PR I am providing a new gtest wrapper header to be used when writing unit tests.

Instead of using directly the gtest header that throws many warnings into the console when we use Visual Studio, we **must** use this wrapper that will skip some warnings that are useless.

### CODE REVIEW

File: unitTests/gtestwrapper.h
Comment (D4N) [inline unitTests/gtestwrapper.h]: Please use ordinary include guards, `#pragma once` is non-standard and while being accepted by gcc & clang, there is no guarantee for that.

**Review by D4N — CHANGES_REQUESTED** [review]

File: unitTests/gtestwrapper.h
Comment (piponazo) [reply unitTests/gtestwrapper.h]: Good point. I tend to use this since most of the most popular compilers implement it (gcc, clang, visual studio, intel), but I did not consider that Exiv2 is used much more extensively than my personal projects

**Comment by piponazo** [comment]
Change done and CI is passing. Waiting for one approval to merge ;)

**Review by tbeu — COMMENTED** [review]
Does the new header gtestwrapper.h appear anywhere in Cmakelist or MSVC solution?

**Review by D4N — APPROVED** [review]

**Comment by piponazo** [comment]
Good point. I think I forgot to include it in the CMake code as another
source. I will add it before merging the branch

**Pull request merged**
